ebl-surrender/src/main/java/org/dcsa/conformance/standards/eblsurrender/party/EblSurrenderCarrier.java

```diff
@@ -82,12 +82,26 @@ private void supplyScenarioParameters(JsonNode actionPrompt) {
     String tdr = UUID.randomUUID().toString().replace("-", "").substring(0, 20);
     eblStatesById.put(tdr, EblSurrenderState.AVAILABLE_FOR_SURRENDER);
     persistentMap.save("response", actionPrompt.get("response"));
+
```

Comment:
Why are these changes part of this PR?

Reply:
Because I updated the standard with the latest schema from the OpenAPI repo. The contain updates, which were not included in the current JSON schema in dev.